### DISCUSSION TITLE
交流贴：Cloudflare的邮件路由可以和其他邮局共存吗

### DISCUSSION THREAD

**Discussion by PIKACHUIM** [dreamhunter2333/cloudflare_temp_email#298] [Q&A]
众所周知，Cloudflare的邮件路由只能
但是托管的邮件路由需要设置MX和SPF记录，托管之后没办法自己修改这俩记录
能不能只用Cloudflare的子域名路由邮件，主域名绑定其他邮局系统

**Comment by dreamhunter2333** [comment]
目前不能共存吧，我没找到怎么设置

**Comment by PIKACHUIM** [comment]
可以共存，提供三种方案
1、使用emailflare，这种办法最方便且有web前端，缺点是不能通过SMTP服务发送，但可以走API发送，接收通过cf路由
2、使用gmail等可以添加第三方发送的邮局，可以走SMTP，缺点是都有限制（发送数量等）
3、自建邮局，先关闭cf的路由，自建邮局配置好了之后，再开启cf路由，解锁添加你自己邮局的spf+cfid+dkim，但不设置mx记录，接收邮件通过cf转发



---原始邮件---
发件人: "Dream ***@***.***&gt;
发送时间: 2023年12月29日(周五) 下午5:04
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [dreamhunter2333/cloudflare_temp_email] 交流贴：Cloudflare的邮件路由可以和其他邮局共存吗 (Issue #53)




 
目前不能共存吧，我没找到怎么设置

**Comment by dreamhunter2333** [comment]
https://github.com/dreamhunter2333/cloudflare_temp_email/releases/tag/v0.2.1 
已添加通过 mailchannels 发送邮件功能
